lib/cog_api/http/base.ex

```diff
@@ -60,7 +60,7 @@ defmodule CogApi.HTTP.Base do
 
   def find_id_by(%Endpoint{}=endpoint, resource, find_fun)
       when is_function(find_fun) do
-    with {:ok, %{^resource => items}} <- get(endpoint, resource) do
+    with {:ok, %{^resource => items}} <- format_response(get(endpoint, resource)) do
```

Comment:
Oh nice, yeah pushing all that response mangling into one function is great. You could even add this to `get` if you wanted.